conftest.py

```diff
@@ -562,14 +562,15 @@ def selenium_common(request, web_server_main, load_pyodide=True):
     """
 
     server_hostname, server_port, server_log = web_server_main
+    cls: type[SeleniumWrapper]
     if request.param == "firefox":
         cls = FirefoxWrapper
     elif request.param == "chrome":
         cls = ChromeWrapper
     elif request.param == "node":
         cls = NodeWrapper
     else:
-        assert False
+        raise AssertionError(f"Unknown browser: {request.param}")
```

Comment:
You shouldn't depend on assert False to catch, assertions could be disabled. Assertions should never "happen", (and) be used to control mypy.

Reply:
Assertions can be disabled?

Reply:
Yes, `python -O` turn off assertions. If you were running pylint, it would balk at this, telling you `assert False` should never be used, and you should raise an AssertionError instead.